packages/components/src/Drawer/Drawer.component.js

```diff
@@ -182,12 +205,27 @@ function Drawer({
 	tabs,
 	withTransition,
 	getComponent,
+	selectedTabKey,
 }) {
 	if (!children) {
 		return null;
 	}
 
 	const TabBarComponent = Inject.get(getComponent, 'TabBar', TabBar);
+
+	let activeTab = {};
+	let activeTabItem = [];
+	let customTabs;
+	if (tabs && tabs.items.length > 0) {
+		customTabs = Object.assign({}, tabs);
+
+		if (selectedTabKey) {
+			Object.assign(customTabs, { selectedKey: selectedTabKey });
```

Comment:
Why didn't you do something like `customTabs.selectedKey = selectedTabKey;` ?

Reply:
i changed it